docs/source/overview/Prerequisites.rst

```diff
@@ -0,0 +1,67 @@
+#############
```

Comment:
Other stupid remark: some files have capitalized names (this and `Scalevar.rst`), but the large majority are lower case as usual.

Standardize all of the to lower case.